client/js/lib/procedure/DPWizard.js

```diff
@@ -103,15 +104,30 @@ export default function DpWizard () {
         return this
       },
 
+      showElement: function ($elements) {
+        const elements = $elements.toArray()
+        elements.forEach((element) => {
+          element.setAttribute('style', 'display: block !important')
+        })
+      },
+
       hide: function () {
         $form.removeClass('o-wizard-mode')
         this.$items.removeClass('o-wizard--active')
         this.$items.find('.o-wizard__content').removeClass('is-active')
-        $wizardElements.hide().attr('aria-hidden', true)
-        $form.find('.o-wizard__btn--next').hide()
+        $wizardElements.attr('aria-hidden', true)
+        this.hideElement($wizardElements)
+        this.hideElement($form.find('.o-wizard__btn--next'))
         return this
       },
 
+      hideElement: function ($elements) {
+        const elements = $elements.toArray()
+        elements.forEach((element) => {
+          element.setAttribute('style', '')
```

Comment:
shouldn't here (and in the showElement-method) still be the `aria-hidden` handling?

Reply:
switching the `aria-hidden` value is untouched by the changes, it's still happening in line 93 and 118.

Reply:
ok. I see. I expected it within the method. But I understand now, that You don't want to set it for other elements than $wizardelements.